docker-compose.yml

```
@@ -21,7 +21,7 @@ services:
       - ./:/home/package
 
   meilisearch:
-    image: getmeili/meilisearch:latest
+    image: getmeili/meilisearch:v1.0.0-rc.3
```

Do we want to keep this specific version? 
```suggestion
    image: getmeili/meilisearch:latest
```